tests/unit/components/DocumentationTopic.spec.js

```diff
@@ -211,33 +212,29 @@ describe('DocumentationTopic', () => {
   it('renders a `DocumentationHero`, enabled', () => {
     const hero = wrapper.find(DocumentationHero);
     expect(hero.exists()).toBe(true);
-    expect(hero.props()).toEqual({ type: propsData.symbolKind, enhanceBackground: true });
-  });
-
-  it('renders a `DocumentationHero`, enabled, with a the `role`, if no symbolKind', () => {
-    wrapper.setProps({
-      role: TopicTypes.article,
-      symbolKind: '',
-    });
-    const hero = wrapper.find(DocumentationHero);
-    expect(hero.props()).toEqual({ type: TopicTypes.article, enhanceBackground: true });
+    expect(hero.props()).toEqual({ type: propsData.role, enhanceBackground: true });
   });
 
   it('render a `DocumentationHero`, enabled, if top-level technology page', () => {
-    wrapper.setProps({
-      role: TopicTypes.collection,
-      symbolKind: 'module',
-    });
     const hero = wrapper.find(DocumentationHero);
-    expect(hero.props()).toEqual({ type: TopicTypes.module, enhanceBackground: true });
+    expect(hero.props()).toEqual({ type: TopicTypes.collection, enhanceBackground: true });
   });
 
   it('render a `DocumentationHero`, disabled, if symbol page', () => {
-    wrapper.setProps({
+    /* wrapper.setProps({
```

Comment:
What is wrong with this? I tried with setProps using the props you set below and it works.

Reply:
I don't know. Maybe I'm doing something dumb, but the test fails with the below patch because it doesn't seem to work on my machine (still has the old props):

```diff
diff --git a/tests/unit/components/DocumentationTopic.spec.js b/tests/unit/components/DocumentationTopic.spec.js
index b94316a..2c0caf5 100644
--- a/tests/unit/components/DocumentationTopic.spec.js
+++ b/tests/unit/components/DocumentationTopic.spec.js
@@ -226,12 +226,16 @@ describe('DocumentationTopic', () => {
     }); */
 
     // setProps isn't working for some reason...
-    wrapper = shallowMount(DocumentationTopic, {
+    /*wrapper = shallowMount(DocumentationTopic, {
       propsData: {
         ...propsData,
         role: 'symbol',
         symbolKind: 'protocol',
       },
+    });*/
+    wrapper.setProps({
+      role: 'symbol',
+      symbolKind: 'protocol',
     });
     const hero = wrapper.find(DocumentationHero);
     expect(hero.props()).toEqual({ type: 'symbol', enhanceBackground: false });
```

Result:
```shell
  ● DocumentationTopic › render a `DocumentationHero`, disabled, if symbol page

    expect(received).toEqual(expected) // deep equality

    - Expected
    + Received

      Object {
        "enhanceBackground": false,
    -   "type": "symbol",
    +   "type": "collection",
      }

      239 |     });
      240 |     const hero = wrapper.find(DocumentationHero);
    > 241 |     expect(hero.props()).toEqual({ type: 'symbol', enhanceBackground: false });
          |                          ^
      242 |   });
      243 |
      244 |   it('renders a `Title`', () => {

      at Object.<anonymous> (tests/unit/components/DocumentationTopic.spec.js:241:26)
```

Reply:
This one works. Note the two props are switched. I have no idea why...

```js
    wrapper.setProps({
      symbolKind: 'protocol',
      role: 'symbol',
    });
    const hero = wrapper.find(DocumentationHero);
    expect(hero.props()).toEqual({ type: 'symbol', enhanceBackground: false });
```

Reply:
Lol, that's a fun bug